Admin.js

```diff
@@ -114,561 +136,771 @@ var Admin = {
+            "error" (xhr, statusText, errorThrown) {
+
                 return xhr.responseText;
+
             }
         });
+
     },
 
     /**
-     * render log message
+     * Render log message
      * @param mixed
      */
-    log: function() {
-        var msg = '[Sonata.Admin] ' + Array.prototype.join.call(arguments,', ');
+    "log" () {
+
+        const msg = `[Sonata.Admin] ${Array.prototype.join.call(arguments, ", ")}`;
         if (window.console && window.console.log) {
+
             window.console.log(msg);
+
         } else if (window.opera && window.opera.postError) {
+
             window.opera.postError(msg);
+
         }
+
     },
 
     /**
      * NEXT_MAJOR: remove this function.
      *
      * @deprecated in version 3.0
      */
-    add_pretty_errors: function() {
-        console.warn('Admin.add_pretty_errors() was deprecated in version 3.0');
+    "add_pretty_errors" () {
+
+        console.warn("Admin.add_pretty_errors() was deprecated in version 3.0");
+
     },
 
-    stopEvent: function(event) {
+    "stopEvent" (event) {
+
         // https://github.com/sonata-project/SonataAdminBundle/issues/151
-        //if it is a standard browser use preventDefault otherwise it is IE then return false
-        if(event.preventDefault) {
+        // If it is a standard browser use preventDefault otherwise it is IE then return false
+        if (event.preventDefault) {
+
             event.preventDefault();
+
         } else {
+
             event.returnValue = false;
+
         }
 
-        //if it is a standard browser get target otherwise it is IE then adapt syntax and get target
-        if (typeof event.target != 'undefined') {
+        // If it is a standard browser get target otherwise it is IE then adapt syntax and get target
+        if (typeof event.target !== "undefined") {
+
             targetElement = event.target;
+
         } else {
+
             targetElement = event.srcElement;
+
         }
 
         return targetElement;
+
     },
 
-    add_filters: function(subject) {
-        Admin.log('[core|add_filters] configure filters on', subject);
-        jQuery('a.sonata-toggle-filter', subject).on('click', function(e) {
+    "add_filters" (subject) {
+
+        Admin.log("[core|add_filters] configure filters on", subject);
+        jQuery("a.sonata-toggle-filter", subject).on("click", (e) => {
+
             e.preventDefault();
             e.stopPropagation();
 
-            if (jQuery(e.target).attr('sonata-filter') == 'false') {
+            if (jQuery(e.target).attr("sonata-filter") == "false") {
+
                 return;
+
             }
 
-            Admin.log('[core|add_filters] handle filter container: ', jQuery(e.target).attr('filter-container'))
+            Admin.log("[core|add_filters] handle filter container: ", jQuery(e.target).attr("filter-container"));
+
+            const filters_container = jQuery(`#${jQuery(e.currentTarget).attr("filter-container")}`);
```

Comment:
- eslint: Line 300 exceeds the maximum line length of 80.
- eslint: Identifier 'filters_container' is not in camel case.